.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: cargo fmt --all -- --check
 
       - name: Build and install cargo-binutils
-        run: cargo install --path . -f
```

Comment:
Testing only against locked deps is bad idea as for me. Like sweeping the trash under the carpet.
We need to see if something fails and fix it.
In this case we just need to choose what to do: lock failing dependency or bump MSRV to 1.73.

Reply:
This typically fails on completely unrelated pull requests, possibly overwhelming a first-time contributor.

For checking that the build still works with the latest versions of some upgraded dependencies, I'd very much prefer a scheduled job running on the master branch over the jobs triggered on pull requests.

Reply:
> Testing only against locked deps is bad idea as for me. Like sweeping the trash under the carpet. We need to see if something fails and fix it. In this case we just need to choose what to do: lock failing dependency or bump MSRV to 1.73.

I suggest we test on stable rust without `--locked` to catch breaks by new versions, but also test the MSRV **with** `--locked` so we can see it still compiles on that Rust even if dependencies have released new versions with bumped MSRV. Does that sound OK? Then we can also add a scheduled CI job to regularly test so that new versions are caught automatically instead of on the next PR.